### DISCUSSION TITLE
4chut89

### DISCUSSION THREAD

**Discussion by 4chut89** [community/community#20271] [Other Feature Feedback, Questions, & Ideas]
Ask a quesion

**Comment by radar-blip** [comment]
Which one?

**Reply by TurboCharged44** [reply]
> Which one?

Skootin4436944@gmail.com

**Comment by Dogl4** [comment]
Which came first?
```js
const first = ["🥚", "🐔"];

console.log(first.sort()); 
```

**Reply by BlyNTC** [reply]
Hello guys!!

**Reply by Atharr** [reply]
The 🥚necessarily came first, because they were around before chickens even existed.

**Comment by Dogl4** [comment]
For gain achievements, look in <https://github.com/Schweinepriester/github-profile-achievements>

**Comment by radar-blip** [comment]
> Ask a quesion

Get an answer!
